recipes/python/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: python
+  version: "3.5.1"
```

Comment:
Sounds reasonable. Is there some discussion of this happening somewhere? It would be nice to iron out how this works before we get a bunch of things like `gcc`, `libgcc`, etc.